hash.go

```diff
@@ -74,10 +70,27 @@ func (h *hasher) trip(t *Trip) {
 func (h *hasher) vehicle(v *Vehicle) {
```

Comment:
In addition to these changes, we likely need a public function to support hashing a vehicle? Something like:
```
func (v *Vehicle) Hash(h hash.Hash) {
	s := hasher{h: h}
	s.vehicle(v)
	s.flush()
}
```

Reply:
I believe this function already exists (`hash.go`, line 23).

Reply:
Woops, yes!